data/sql/updates/pending_db_world/rev_1611865917828980400.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1611865917828980400');
+INSERT INTO item_template_locale (ID, locale, Name, Description, VerifiedBuild) VALUES ('30622', 'zhCN', '焰铸钥匙', '允许你进入英雄难度的地狱火堡垒地下城。', '15050'); 
+INSERT INTO item_template_locale (ID, locale, Name, Description, VerifiedBuild) VALUES ('30637', 'zhCN', '焰铸钥匙', '允许你进入英雄难度的地狱火堡垒地下城。', '15050');
```

Comment:
Nice, this is better, now I will teach you something =)

1. Always use DELETE before INSERT
2. You can use 1 INSERT here
3. always use \` (backticks) around SQL keywords
4. Never use \'\' around numbers
5. and add a newline ;)

Reply:
I tried ;)